docs/setup.md

```diff
@@ -1,44 +1,61 @@
 | [Home](../README.md) |
-|--------------------------------------------|
+|----------------------|
 
 # Installation
 
 1. To install a widget, click **Content Hub** > **Discover**.
 
-2. From the list of solution pack that appears, search for and select **Continuous Delivery**. 
+2. From the list of widgets that appears, search for and select **CICD Configuration Wizard**. 
 
-3. Click the card of the **Continuous Delivery** solution pack card.
+3. Click the card of the **CICD Configuration Wizard** widget card.
 
-4. Click **Configure** on the bottom to begin configuration.
+4. Click **Install** on the bottom to begin installation.
 
 # Configuration
 
 ## CICD Configuration Wizard Widget Settings
 
 The configuration settings of the CICD Configuration widget include the following steps to setup CICD configuration:
 
-- Click the **Configure** button to bring up the following wizard screen
+1. Click the **Configure** button to bring up the following wizard screen.
```

Comment:
@cs-abhishek-shukla : Before this, point we should give the details from where user can click configure button.

Reply:
Added